HW02/bright_okike/hw02.rb

```diff
@@ -0,0 +1,70 @@
+module My_enumerables
```

Comment:
```suggestion
module MyEnumerables
```
Could you also change the file name please?